### PR TITLE
Fix typos and formatting

### PR DESCRIPTION
**Pull request by friederbluemle**
Just a bunch of typo and whitespace fixes.

### CODE REVIEW

File: system-tests/fixtures/api/ComplexApi/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "author": "blemair",
+  "author": "f0b00n7",
```
Comment (krunalsshah) [inline system-tests/fixtures/api/ComplexApi/package.json]: pls, check if this is intentional. IMHO, none is correct, is CI workspace set up as `/Users/blemair/Code/`?

File: system-tests/fixtures/api/ComplexApi/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "message": {
       "moduleName": "ComplexApi",
       "namespace": "com.ComplexApi.ern",
-      "apiSchemaPath": "/Users/blemair/Code/electrode-native/system-tests/fixtures/api/complexapi-schema.json",
+      "apiSchemaPath": "/Users/f0b00n7/github/electrode-io/electrode-native/system-tests/fixtures/api/complexapi-schema.json",
```
Comment (krunalsshah) [inline system-tests/fixtures/api/ComplexApi/package.json]: pls, check if this is intentional. IMHO, none is correct, is CI workspace set up as /Users/blemair/Code/?

File: system-tests/fixtures/api/TestApi/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "message": {
       "moduleName": "TestApi",
       "namespace": "com.TestApi.ern",
-      "apiSchemaPath": "/Users/blemair/Code/electrode-native/system-tests/fixtures/api/testapi-schema.json",
+      "apiSchemaPath": "/Users/f0b00n7/github/electrode-io/electrode-native/system-tests/fixtures/api/testapi-schema.json",
```
Comment (krunalsshah) [inline system-tests/fixtures/api/TestApi/package.json]: pls, check if this is intentional. IMHO, none is correct, is CI workspace set up as /Users/blemair/Code/?

File: system-tests/fixtures/api/ComplexApi/package.json (hunk 2 above)
Comment (friederbluemle) [reply system-tests/fixtures/api/ComplexApi/package.json]: These files are auto-generated. It's not "intentional" ;)

File: system-tests/fixtures/api/TestApi/package.json (hunk 3 above)
Comment (friederbluemle) [reply system-tests/fixtures/api/TestApi/package.json]: Yeah, I also don't like this. Check #1301

**Pull request merged**